agito.py

```diff
@@ -39,6 +39,10 @@
 MERGEINFO_LINE_RE = re.compile(r'([\w/\-]+):.*-(\d+)')
 WORD_MATCH = re.compile(r'(\s*)(\S+)')
 BULLET_POINT_RE = re.compile(r'(\s*[\*\+]\s*)')
+COMMIT_REF_REs = (
+	re.compile(r"r([0-9.]+)"),
```

Comment:
Huh, you're right. TIL.

    >>> import re
    >>> MYRE = re.compile('r[0-9.]+')
    >>> MYRE.match('r0131.23asdf').group(0)
    'r0131.23'